### DISCUSSION TITLE
Define sort order on associated modules records

### DISCUSSION THREAD

**Discussion by yeti92force** [YetiForceCompany/YetiForceCRM#16734] [How to] [answered]
Hi dear support,

I would like modify the default sort order of my related list records
![image](https://user-images.githubusercontent.com/107400809/207319766-53082c61-7d29-4007-8064-54e7a3ad4927.png)

Anyway, applying a filter is not helping.
I'm trying all my best on YetiForce 6.4 and 5.3.

nb : 
![image](https://user-images.githubusercontent.com/107400809/207320137-9d195701-73cc-480a-a2b0-0e594e520ff6.png)
i can define a sort order by this way on the parent record resume however change are not saved. Its just working once

Thanks!
Regards,
Mathieu Skz

**Comment by rskrzypczak** [comment]
Permanent sorting for filters is set in the configuration panel:
![image](https://user-images.githubusercontent.com/10221999/207323703-845627ff-2fab-4e66-a931-e03b872bf250.png)

The list of related records generally does not include sorting.

**Reply by yeti92force** [reply]
I would like modify the default sort order of my related list records (on a date field). Applying a sort order on Filters configuration is not affecting related list records order

**Comment by rskrzypczak** [comment, marked as answer]
You can set sort order directly in the file, but this applies globally: https://github.com/YetiForceCompany/YetiForceCRM/issues/1572

**Reply by yeti92force** [reply]
its working! thank guy, you rock
